reconcile/change_owners/change_log_tracking.py

```diff
@@ -157,7 +157,9 @@ def run(
                         changed_apps = {
                             name
                             for c in change_versions
-                            if (name := app_name_by_path.get(c["app"]["$ref"]))
+                            if (app := c["app"])
```

Comment:
would `c.get("app")` make sense here too?

Reply:
`app` is always there, but the object inside can differ.